File: 283-move-zeroes/283-move-zeroes.java
class Solution {
    public void moveZeroes(int[] nums) {
        int zeroRemind=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]!=0){
                nums[zeroRemind]=nums[i];
                zeroRemind++;
            }    
        }
        for(int i=zeroRemind;i<nums.length;i++){
            nums[i]=0;
        }
        
    }
}

File: 283-move-zeroes/283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        zeroRemind=0
        for x in range(len(nums)):
            if nums[x]!=0:
                nums[zeroRemind]=nums[x]
                zeroRemind+=1
        for x in range(zeroRemind,len(nums)):
            nums[x]=0